lib/fluent/plugin_helper/server.rb

```diff
@@ -355,7 +355,11 @@ def server_create_tcp_socket(shared, bind, port)
         sock = if shared
                  server_socket_manager_client.listen_tcp(bind, port)
                else
-                 TCPServer.new(bind, port) # this method call can create sockets for AF_INET6
+                 # TCPServer.new doesn't set IPV6_V6ONLY flag, so use Addrinfo class instead.
+                 # backlog will be set correctly so use listen's default here
```

Comment:
> backlog will be set correctly

What does this mean?  who sets backlog?

Reply:
`server_create_tcp_socket` is used in other API, e.g. server_create, so it sets backlog from plugins.

Reply:
This one? https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin_helper/server.rb#L222
How about "backlog will be set by the caller, we don't set backlog here", then? (it's IMO. so it's ok for me to leave as it is)